domains/nairoby.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "nairobylamocana",
+        "email": "nairobylamocana@gmail.com"
+    },
+    "records": {
+        "CNAME": "alaric.ns.cloudflare.com"
+        "CNAME": "dalary.ns.cloudflare.com"
```

please use NS records for this and also state your reasoning for needing NS records
```suggestion
        "NS": ["alaric.ns.cloudflare.com", "dalary.ns.cloudflare.com"]
```